### DISCUSSION TITLE
당근마켓 최종합격까지 - 정현수 기술 블로그

### DISCUSSION THREAD

**Discussion by giscus** [junghyeonsu/junghyeonsu.dev.comment#30] [Announcements]
# 당근마켓 최종합격까지 - 정현수 기술 블로그

당근마켓 썸머테크 인턴에서 시작해서 정규직 전환까지

https://junghyeonsu-dev.vercel.app/posts/daangn-pass-retrospect

**Comment by irrationnelle** [comment]
> 내가 나중에 이 글을 봤을 때 아, 이런 감정을 느꼈었지 지금은 어떻고 그 때는 어땠는데 하면서 회상할 수 있기를

이거 정말 공감해요. 그래서 글로 남기는 게 정말 좋은 거 같아요. 저도 첫회사에서 첫커밋 넣었을 때 벅찬 기분이 있었는데 글로 남기지 않아서 그런지 기억이 희미하거든요.

최종 합격 축하드려요~!

**Comment by ginger-kang** [comment]
공감되는 부분이 많네요ㅋㅋㅋ 너무 고생하셨고 축하드려용 🎉

**Comment by jiseung-kang** [comment]
정말 정신적으로도, 체력적으로도 소모가 너무나 컸을 인턴기간을 너무너무너무너무 잘 이겨내고 결실을 얻게 된 걸 진심으로 축하해요!!🥳 달라진 건 크게 없다는 말.. 대인배의 느낌이 물씬 나버려 ,,

이번을 시작으로 앞으로도 개발자 인생 다이나믹하게(?!) 살아갈 텐데, 지금의 경험을 디딤돌 삼아 뭐든 하고 싶은 것들, 이루고 싶은 것들 항상 도전하고 또 성취할 수 있기를 바라고 응원해요~!~! 💪💪 

(연말 회고도 기다릴게요..

**Reply by junghyeonsu** [reply]
감사해요 춘티........... ㅜㅜㅜㅜㅜㅜㅜㅜㅜㅜㅜㅜ

**Comment by tooooo1** [comment]
새로운 여행을 시작하셨네요! ㅎㅎ 잘되실줄 믿었어요 ㅎㅎ 앞으로도 멋진 행보 옆에서 응원할게요!

**Comment by solidw** [comment]
최종합격 축하드려요. 대서사시의 시작을 응원합니다!
